lib/select.js

```diff
@@ -2797,8 +2797,12 @@ global.select = factory();
 
     if (element.hasAttribute(sSearchEmptyAtt)) {
       obj.noResultsText = element.getAttribute(sSearchEmptyAtt);
+    } else {
+      obj.noResultsText = vl.i18n.t('select.no_results');
     }
 
+    obj.noChoicesText = vl.i18n.t('select.no_more_options');
```

Comment:
Waarom heeft select.no_results wel een if-test er rond staan met een check op een attribuut en deze regel niet?

Reply:
Omdat de select.no_results ook nog met het data-vl-select-search-empty-text attribuut kan gezet worden (AIV code). Voor de select.no_results hebben ze niets voorzien (of althans, ik heb er niets  voor gevonden)

Reply:
Ok. Ja, 't is verwarrend op den duur :)

Reply:
@SanderKleykens en @wautergu nemen we hier nog actie of kan deze conversatie resolved worden?

Reply:
Nee voro mij kan hij resolved worden